Mlem/App/Views/Pages/PersonView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  PersonView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 30/05/2024.
+//
+
+import LemmyMarkdownUI
+import MlemMiddleware
+import SwiftUI
+
+struct PersonView: View {
+    enum Tab: String, CaseIterable, Identifiable {
+        case overview, comments, posts, communities
+
+        var id: Self { self }
+        var label: String { rawValue.capitalized }
+    }
+    
+    @State var person: AnyPerson
+    @State var selectedTab: Tab = .overview
+    @State var isAtTop: Bool = true
+    
+    var body: some View {
+        ContentLoader(model: person) { person in
+            FancyScrollView(isAtTop: $isAtTop) {
+                VStack(spacing: AppConstants.standardSpacing) {
+                    ProfileHeaderView(person, type: .person)
+                        .padding(.horizontal, AppConstants.standardSpacing)
+                    bio(person: person)
+                    if let person = person as? any Person3Providing {
+                        VStack(spacing: 0) {
+                            personContent(person: person)
+                        }
+                        .transition(.opacity)
+                    } else {
+                        VStack(spacing: 0) {
+                            Divider()
+                            ProgressView()
+                                .padding(.top)
+                        }
+                        .transition(.opacity)
+                    }
+                }
+                .animation(.easeOut(duration: 0.2), value: person is any Person3Providing)
+            }
+            .navigationTitle(isAtTop ? "" : (person.displayName_ ?? person.name))
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .topBarTrailing) {
+                    // TODO:
+                    ToolbarEllipsisMenu {}
+                }
+            }
+        }
```

Comment:
Extracting this to a `content()` function makes the `ContentLoader` structure a little prettier